### ISSUE TITLE
Tooltips with Power Bi Embedded don't update

### ISSUE THREAD

**Issue by carlhodder** [mapbox/mapboxgl-powerbi#311]
I have a potential issue with mapbox when embedded as part of Power BI: The tooltips don't update.

When a slicer is changed to limit the data, the map zooms/refreshes but the tooltips don't change. e.g. I can change the slicer from '2018' to '2002', have the map auto-zoom but still have tooptip data for '2018'.

The map works fine in Power BI desktop and when viewing it through the power BI portal, just not as an embed.

Currently can't expose this as an example as it has private data, so I'll try to mock an example up over the weekend. However I see you're switching your tooltip approach in the beta, so this may not be needed at all?

Cheers,
Carl

**Comment by ryanbaumann** [comment]
Thanks for the report @carlhodder.

> Currently can't expose this as an example as it has private data, so I'll try to mock an example up over the weekend. 

Thank you!  That will be needed information for us to reproduce this bug.

> However I see you're switching your tooltip approach in the beta, so this may not be needed at all?

Can you link to the ticket or PR indicating the change to tooltips you're referring to?

cc/ @samgehret

**Comment by carlhodder** [comment]
My apologies for this - the fault is all mine. I'll blame lack of sleep but it's so stupid I should've spotted it debugging - the JS code we've got around the slicers had been modified and someone cast the data to a string when formatting API calls.

The target data type is an integer... Sorry!

I've tried to find the ticket about tooltips that mentioned altering the tooltips in 'new beta' but I can't spot exactly what I remember reading, it could've been crossed wires with #270 or #249.


Once again, apologies.
